packages/server/graphql/types/User.ts

```diff
@@ -399,6 +400,40 @@ const User = new GraphQLObjectType<any, GQLContext>({
         return source.overLimitCopy
       }
     },
+    similarReflectionGroups: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(RetroReflectionGroup))),
+      description:
+        'The reflection groups that are similar to the selected reflection in the Spotlight',
+      args: {
+        reflectionId: {
+          type: GraphQLID,
```

Comment:
When the Spotlight closes, the `spotlightReflectionId` becomes null and the query in the `SpotlightRoot` fires. To make `reflectionId` nonnullable, we could use [useLazyLoadQuery](https://relay.dev/docs/next/api-reference/use-lazy-load-query/) instead of the `QueryRenderer` and pass in the `reflectionId` as the skip variable?